packages/create-flex-plugin/src/lib/cli.ts

```diff
@@ -73,6 +74,14 @@ export default class CLI {
     version: {
       alias: 'v',
     },
+    flexui1: {
```

Comment:
You want to add `exclusive` flag to both flexui1 and flexui2 because you should not be able to provide both at the same time. See https://oclif.io/docs/flags